extensions/amp-form/0.1/amp-form.js

```diff
@@ -216,6 +221,29 @@ export class AmpForm {
   }
 
   /**
+   * Returns form data as an object.
+   * @return {!Object}
+   * @private
+   */
+  getFormAsObject_() {
+    const data = {};
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
```

Comment:
It might be faster to use `#elements`, then filter [like jQuery](https://github.com/ampproject/amphtml/pull/4562/files#r75363998).